Automattic/pocket-casts-android
# Integrate PlaybackManager more tightly into wear app

## Description
This PR improves how the wear app integrates with our `PlaybackManager`.

I think we want to lean heavily on our `PlaybackManager` class because that is (1) easier since we've already written it and (2) gives us consistent behavior between the watch and phone apps for free. Horologist, however, has a `PlayerRepository` class (and some related classes that use the `PlayerRepository`) that largely replicate the functionality of our `PlaybackManager`. We were trying to use the `PlayerRepository` before this PR, but I think we're better off not using that class and instead directly integrating our `PlaybackManager`, which is what this PR does.

The work on playback isn't done with this PR (but it's a step in the right direction imo), so please let me know any feedback/thoughts you have. I'm sure we'll be going through a couple more iterations on this before we find the right architecture here.

## Testing Instructions
1. Install the wear app, and sign in to the app
2. Observe that the Up Next and Now Playing chips are updated appropriately based on the account info.
3. Go to the Now Playing screen.
4. Verify that play/pause works
5. verify that the skip forward/backward buttons work (and that the skip amounts reflect any custom skip settings on the account)
6. Go to the Up next screen
7. Tap on an episode and verify that the watch navigates to the Now Playing screen and begins playing that episode.
8. Swipe back to the up next queue and verify that the episode that was playing before is now at the top of the up next queue.
9. Open the podcasts screen and tap on one of the podcasts
10. Scroll down and select an episode that is not queued
11. Observe that the now playing screen is opened and the selected episode starts playing
12. Verify that the previously playing episode is now at the top of the up next queue.

## Checklist
